src/test/java/org/mifos/integrationtest/cucumber/stepdef/VoucherManagementStepDef.java

```diff
@@ -462,4 +462,41 @@ public void iWillAssertTheFieldsFromFetchVoucherResponse() {
             logger.debug(e.getMessage());
         }
     }
+
+    @Given("I can create an negative VoucherRequestDTO for voucher creation")
+    public void createNegativeVoucherRequestDTO() {
+        requestId = generateUniqueNumber(12);
+        StringBuilder sb = new StringBuilder();
+        sb.append("{\n");
+        sb.append("    \"requestID\": \"").append(requestId).append("\",\n");
```

Comment:
Do not create the request body using String, create a POJO and then convert that to JSON. We did this approach earlier and ended up with loads of technical debt.